cmd/mock-driver/main.go

```diff
@@ -28,8 +28,17 @@ import (
 	"github.com/kubernetes-csi/csi-test/v4/driver"
 	"github.com/kubernetes-csi/csi-test/v4/mock/service"
 	"gopkg.in/yaml.v2"
+	"k8s.io/klog"
 )
 
+func init() {
+	// klog verbosity guide for this package
+	// Use V(2) for one time config information
+	// Use V(3) for general debug information logging
+	klog.InitFlags(flag.CommandLine)
+	flag.Set("logtostderr", "true")
```

Comment:
Setting logtostderr should be redundant: https://github.com/kubernetes/klog/blob/4f12f780d4781eb828bc6520ae87d7516f30f0b2/klog.go#L38-L40

Can you remove this line and double-check?